src/test/resources/authority/100/marc_100_concept_Ind1_equals3.jsonl

```diff
@@ -1,6 +1,7 @@
 {
   "fields" : [ {
-    "001":"sh85121033"
+    "001":"sh85121033",
+    "010":"010 field value"
```

Comment:
same comment here. "010" is not a control field. It is a data field.

Reply:
done.